fe/fe-core/src/main/java/com/starrocks/fs/hdfs/HdfsFsManager.java

```diff
@@ -486,33 +499,43 @@ public FileSystem run() throws Exception {
     public HdfsFs getS3AFileSystem(String path, Map<String, String> loadProperties, THdfsProperties tProperties)
             throws UserException {
         WildcardURI pathUri = new WildcardURI(path);
+        HdfsFsIdentity fileSystemIdentity = null;
+
+        CloudConfiguration cloudConfiguration =
+                CloudConfigurationFactory.buildStorageCloudConfiguration(loadProperties);
+
         String accessKey = loadProperties.getOrDefault(FS_S3A_ACCESS_KEY, "");
         String secretKey = loadProperties.getOrDefault(FS_S3A_SECRET_KEY, "");
         String endpoint = loadProperties.getOrDefault(FS_S3A_ENDPOINT, "");
         String disableCache = loadProperties.getOrDefault(FS_S3A_IMPL_DISABLE_CACHE, "true");
         String connectionSSLEnabled = loadProperties.getOrDefault(FS_S3A_CONNECTION_SSL_ENABLED, "false");
 
-        if (accessKey.equals("")) {
-            LOG.warn("Invalid load_properties, S3 must provide access_key");
-            throw new UserException("Invalid load_properties, S3 must provide access_key");
-        }
-        if (secretKey.equals("")) {
-            LOG.warn("Invalid load_properties, S3 must provide secret_key");
-            throw new UserException("Invalid load_properties, S3 must provide secret_key");
-        }
-        if (endpoint.equals("")) {
-            LOG.warn("Invalid load_properties, S3 must provide endpoint");
-            throw new UserException("Invalid load_properties, S3 must provide endpoint");
+        if (cloudConfiguration != null) {
```

Comment:
In which case, cloudConfiguration is null?

Reply:
If no new parameters, it will use old logic.